test/test-poll-oob.c

```diff
@@ -0,0 +1,164 @@
+/* Copyright Fedor Indutny & CurlyMo All rights reserved.
```

Comment:
I think we prefer `Copyright libuv project contributors.` for new files nowadays.